tdrs-backend/plg/tempo/manifest.yml

```diff
@@ -0,0 +1,28 @@
+version: 1
+applications:
+  - name: tempo
+    memory: 512M
+    disk_quota: 7G
+    instances: 1
+    command: |
+      mkdir /home/vcap/app/tempo-data
+      S3_URI=$(echo $VCAP_SERVICES | jq -r .s3[0].credentials.uri)
```

Comment:
I think we should start to create a utility function, pull the VCAP env variable in ONE place, change it to api or some other format that doesn't have dependency on JQ everywhere. The way we are using JQ, if for whatever reason JQ package is changed or the format of VCAP env var is changed, then we have to refactor 100 different places.

I think we need a python script, change the json env to regular string and get done with it. I don't like to be dependent on JQ